### DISCUSSION TITLE
Hello! Can someone help me?

### DISCUSSION THREAD

**Discussion by Boji2005** [RedShyGuy/Vapecord-ACNL-Plugin#79] [Q&A] [answered]
I installed the pluggin form the universal updater and enabled the plugin loader. But when I start the game it says: Outdated plugin file 

Error code: 0xD8E07402

**Comment by RedShyGuy** [comment, marked as answer]
Delete any outdated plugin files

**Reply by Boji2005** [reply]
Thanks for the response! Where can I find them?

**Reply by Boji2005** [reply]
Nvm I think I found them. They are folders that are named with a bunch of numbers right?

**Reply by RedShyGuy** [reply]
Yes inside those folders, delete the `.3gx` files that are not named vapecord

**Reply by Boji2005** [reply]
All of the .3gx files are named Vapecord_public and there aren't any other 3gx files.

**Reply by RedShyGuy** [reply]
Ah then your boot.firm is incorrect, make sure to download the correct one from the zip and replace the old one

**Reply by RedShyGuy** [reply]
You could try to redo the download of the plugin

**Reply by Boji2005** [reply]
Oh ok I'll try that later. Thanks for the Help!!

**Reply by Boji2005** [reply]
So the error is gone! But the cheat menu doesn't show up when I press the select button.

Thanks for the help I appreciate it a ton!
